wasm/typescript/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@itk-wasm/morphological-contour-interpolation",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
I wanted to suggest also raising Python package version, but I now see it is already at version [1.1.0](https://github.com/KitwareMedical/ITKMorphologicalContourInterpolation/blob/d4df8ffd86b6994065d3e2788a110c331f423f6d/setup.py#L16).

Reply:
The published Python package is synchronized to the JavaScript package version during publication.